cmd/atlantis_generate_repo_config.go

```diff
@@ -36,35 +36,35 @@ func init() {
 
 	atlantisGenerateRepoConfigCmd.PersistentFlags().String("stacks", "",
 		"Generate Atlantis projects for the specified stacks only (comma-separated values).\n"+
-			"atmos atlantis generate repo-config --config-template <config_template> --project-template <project_template> --stacks <stack1>,<stack2>\n"+
+			"atmos atlantis generate repo-config --config-template &ltconfig_template&gt --project-template &ltproject_template&gt --stacks &ltstack1&gt,&ltstack2&gt\n"+
```

Comment:
This should absolutely not be necessary in markdown.

Reply:
Unfortunately it is being consumed by markdown. It considers anything inside angular brackets as if it is html.

Reply:
> Unfortunately it is being consumed by markdown. It considers anything inside angular brackets as if it is html. 

But then we still have the wrong conclusion.

The problem is that we are not using backticks and code fences.